doc/man3/DH_generate_parameters_ex.pod

```diff
@@ -142,7 +142,7 @@ function.
 DH_generate_parameters_ex(), DH_check() and DH_check_params() return 1
 if the check could be performed, 0 otherwise.
 
-DH_generate_parameters() returns a pointer to the DH structure or NULL if
+DH_generate_parameters_ex() returns a pointer to the DH structure or NULL if
```

Comment:
```suggestion
DH_generate_parameters() returns a pointer to the DH structure or NULL if
```

This needs to remain DH_generate_parameters() since the _ex version returns an int and populates a DH returned by DH_new.

Reply:
This also still needs fixing